api/api.go

```diff
@@ -161,6 +161,12 @@ func (api *API) setupRoutes() {
 	// V1 API
 	v1 := api.r.Group("/api/v1")
 
+	// external checks used by services like load balances
+	externalChecksProtected := v1.Group("/external-checks")
+	{
+		externalChecksProtected.GET("/systems-check", api.ExternalSystemsCheck)
```

Comment:
why not:

```
/external/systems/check
```

this allows better REST expandability:

```
/external/api/check
/external/systems/check
/external/systems/list
```

etc.

Reply:
or even better:

```
/status/systems/check
/status/systems/...
/status/...
```

etc

Reply:
I feel a lot of the other APIs can be collapsed this way as well, to reduce the number of subroutes and have better grouping, though since that will likely cause compatibility havoc it's not something particularly urgent. I'll open up a ticket tho